### DISCUSSION TITLE
Scrape PDFs / News Articles / Favorite Youtube Channel

### DISCUSSION THREAD

**Discussion by keesfluitman** [danielmiessler/Fabric#1019] [Ideas]
Has anyone worked on actually making Fabric useful?
Cron jobs for automation of long PDF files, for instance governmental Rapports or News.
Scraping and summarizing of your Favorite news paper.
Summarize new youtube videos of your favorite youtuber.

(Of course a GUI for easier manual Automation, if I may call it that)

all that outputting to a file at a certain location? I could imagine going to work, opening my Tablet, and instantly checking today's new updates. It would allow me situations like:
Select certain Sources for different news reports, have it summarize the different papers in one file on a certain subject.
Always be up to date on actual governmental Rapports instead of relying on newspapers for that.

**Comment by mseigal** [comment]
I use it on my always on raspberry pi. I have a Siri shortcut that pushes links from my phone to fabric with various custom patterns I have adapted. The results are added as a new note to Obsidian. For example, I will send a yt video tutorial as a shared link. I have a custom pattern called document_this that rewrites the article into a markdown document with suitable headings as if it was a step by step software tutorial. I still want to curate the content I consume or I fear there will be omissions or distortions, but it would be nice if all possible items of interest was dumped into a text file that was summarised. I think you could achieve this with calibre for news content and have a watched file or use email to trigger a fabric automation.

**Reply by mardlive** [reply]
I was trying to make a pattern just like your document_this, would you mind sharing it, or giving me some pointers? I'm fairly new to prompt engineering and was a bit overwhelmed on where to begin or what to use as a baseline.

**Reply by mseigal** [reply]
Here is my pattern. 

# IDENTITY and PURPOSE

You are a technical author who writes detailed non fiction textbooks. You refine the input text to enhance clarity, coherence, grammar, and style.

# Steps

- Analyze all of the input text for grammatical errors, stylistic inconsistencies, clarity issues, and coherence.
- Apply corrections and improvements directly to the text.
- Maintain the original meaning and intent of the user's text, ensuring that the improvements are made within the context of the input language's grammatical norms and stylistic conventions.

# OUTPUT INSTRUCTIONS

- First line is title in less than 10 words   
- Refined and improved text that has no grammar mistakes.
- Output is in formal prose in the style of a textbook aimed at high school students.
- Write in third person and not first person.
- Include NO additional commentary or explanation in the response.
- Each section has a title in markdown heading 2 as ##.
- Each idea or theme is a separate title.
- You write in detailed markdown bullet points. Titles are not numbered.
- You do not lie or make up any information.
- You write using only UK spellings.
- You do not leave out any information,just remove filler words and make the output read like a well written textbook.
- Documentation must include information from all of the input text.


# INPUT:

INPUT:

**Reply by mardlive** [reply]
Thank you so much, that's an amazing starting point for what I was looking to do! 🤗
